src/web_crawler/twir_crawler.rs

```diff
@@ -138,14 +144,18 @@ impl TwirCrawler {
             found_resources.append(&mut self.parse_page(&issue.link, sentence).await?);
 
             if index > limit {
+                error!("The search stopped due to reaching the limit");
```

Comment:
This is not an error, it is intended. Can be added as trace, and please include the last parsed issue

Reply:
Info also works